### ISSUE TITLE
Can't use swagger with aiohttp-transmute

### ISSUE THREAD

**Issue by xunto** [toumorokoshi/transmute-core#11]
@toumorokoshi 

Hi, I found a problem and I think that it's related to the core.

I have this code:
```
@aiohttp_transmute.describe(
    methods="POST",
    paths="/login"
)
async def handle_login(request, auth_data: AuthRequest) -> AuthResponse:
    username: str = auth_data.username
    password: str = auth_data.password
    ...
    return AuthResponse(token=token)
```
which compiles to this:
```
{
    "name": "auth_data",
    "required": true,
    "in": "body",
    "schema": {
        "properties": {
            "username": {
                "type": "string"
            },
            "password": {
                "type": "string"
            }
        },
        "type": "object",
        "title": "AuthRequest"
    }

}
```

The problem is that they are different.  I can't send request using swagger ui (and client api generated using swagger too), because transmute wants:
```
{
  "auth_data": {
    "username": "string",
    "password": "string"
  }
}
```
instead of
```
{
  "username": "string",
  "password": "string"
}
```

And I should point that swagger doesn't allow more than one property with  "in": "body" for one route.

**Comment by toumorokoshi** [comment]
oops, thanks for pointing that out!

I'm sketching up a fix. I should be able to roll that out soon.

**Comment by toumorokoshi** [comment]
@xunto fixed with release 1.1.1. Unit test updated here: https://github.com/toumorokoshi/transmute-core/blob/dddfd1eead80902bb17884d7b27681af889d2be3/transmute_core/tests/swagger/test_function.py#L25
